Makefile.toml

```diff
@@ -1401,14 +1401,28 @@ script = [
     '''
 ]
 
-# This task will clear all tests and resources from the testsys cluster.
+# This task will clear all tests from the testsys cluster.
+# To delete all passed tests use `cargo make clean-test --passed`
+# To delete all failed tests use `cargo make clean-test --failed`
+# To delete all incomplete tests use `cargo make clean-test --running`
 [tasks.clean-test]
 dependencies = ["test-tools"]
 script = [
     '''
     set -eu
     export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
-    testsys ${CARGO_MAKE_TESTSYS_ARGS} delete
+    testsys ${CARGO_MAKE_TESTSYS_ARGS} delete --test ${@}
+    '''
+]
+
+# This task will clear all tests and resources from the testsys cluster.
+[tasks.clean-test-all]
+dependencies = ["test-tools"]
+script = [
+    '''
+    set -eu
+    export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+    testsys ${CARGO_MAKE_TESTSYS_ARGS} delete ${@}
```

Comment:
The existing `Makefile.toml` target convention is that `clean-x` targets do a full clean of the "x" resource.

The overall intent is that `cargo make clean` will put the repo into a state that's pretty close to a fresh checkout. The exceptions are things like the cache for Go modules and Rust crates.

It looks like the goal here is to avoid a full `testsys delete` unless a more extreme reset is wanted.

I'd suggest something like this:
* add `clean-test` to the overall `clean` task, so that `cargo make clean` also cleans up passed and failed tests by default
* rename `clean-test-all` to `purge-test` or `reset-test`
* add an overall `purge` or `reset` target, so that `cargo make purge` does a full clean and then additionally removes the clusters, and perhaps cleans up the Go / Rust caches